assembl/static2/js/app/pages/brightMirrorFiction.jsx

```diff
@@ -20,19 +20,21 @@ import { displayAlert } from '../utils/utilityManager';
 import { getConnectedUserId } from '../utils/globalFunctions';
 import Permissions, { connectedUserCan } from '../utils/permissions';
 // Constant imports
-import { FICTION_DELETE_CALLBACK } from '../constants';
+import { FICTION_DELETE_CALLBACK, PublicationStates } from '../constants';
 // Type imports
 import type { CircleAvatarProps } from '../components/debate/brightMirror/circleAvatar';
 import type { FictionHeaderProps } from '../components/debate/brightMirror/fictionHeader';
 import type { FictionToolbarProps } from '../components/debate/brightMirror/fictionToolbar';
 import type { FictionBodyProps } from '../components/debate/brightMirror/fictionBody';
+
 // Define types
 export type BrightMirrorFictionData = {
   /** Fiction object formatted through GraphQL  */
   fiction: BrightMirrorFictionFragment,
   /** GraphQL error object used to handle fetching errors */
   error: any
 };
+
 export type BrightMirrorFictionProps = {
```

Comment:
Only Props

Reply:
@amensouissi  It s a props, it s the object Data defined in the props and required for flow